modules/tasks/tasks/src/main/java/org/motechproject/tasks/validation/TaskDataProviderValidator.java

```diff
@@ -32,17 +32,17 @@ private TaskDataProviderValidator() {
      * @param provider  the data provider for validation, not null
      * @return  the set of encountered errors
      */
+    public static boolean validateIsNotEmpty(TaskDataProvider provider) {
+        return !checkEmpty(new HashSet<>(), TASK_DATA_PROVIDER, "objects", provider.getObjects());
```

Comment:
It doesn't seem to make sense to expose this method. After all, we only check if provider.getObjects() is empty and return either true or false. We don't use generated errors later on, so this can be completely removed.